src/main/java/bio/terra/app/configuration/ElasticSearchConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package bio.terra.app.configuration;
+
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ElasticSearchConfiguration {
+    @Bean
+    public RestClientBuilder restClientBuilder(
+            @Value("${elasticsearch.hostname}") String hostname,
+            @Value("${elasticsearch.port}") int port
```

Comment:
If you use `@Configuration(prefix = "elasticsearch")` above  I think you change this to be
```suggestion
            @Value("${hostname}") String hostname,
            @Value("${port}") int port
```

This would also help us to only put `elasticsearch` configs in this class.

Reply:
I'll fix this next PR